bindings/girlmath-js/src/lib.rs

```diff
@@ -15,4 +9,96 @@ pub mod exchange;
 #[wasm_bindgen(start)]
 pub fn start() {
     console_error_panic_hook::set_once();
-}
\ No newline at end of file
+}
+
+/// Converts a major currency unit (like dollars) to a minor currency unit (like cents).
+///
+/// # Arguments
+/// * `amt` - A string slice that holds the amount in major currency units.
+/// * `currency_code` - The ISO or cryptocurrency code as a string slice.
+///
+/// # Returns
+/// The amount in minor units
+///
+/// # Examples
+/// ```js
+/// import girlmath from '@tbd54566975/girlmath'
+///
+/// const cents = girlmath.majorToMinor("10.50", "USD")
+/// console.assert("1050" === cents)
+/// ```
+#[wasm_bindgen(js_name = majorToMinor)]
+pub fn major_to_minor_wasm(amt: &str, currency_code: &str) -> Result<String, JsError> {
+    girlmath::major_to_minor(amt, currency_code).map_err(|e| JsError::new(&e))
+}
+
+/// Converts a minor currency unit (like cents) to a major currency unit (like dollars).
+///
+/// # Arguments
+/// * `amt` - A string slice that holds the amount in minor currency units.
+/// * `currency_code` - The ISO or cryptocurrency code as a string slice.
+///
+/// # Returns
+/// The amount in major units
+///
+/// # Examples
+/// ```js
+/// import girlmath from '@tbd54566975/girlmath'
+///
+/// const dollars = girlmath.minorToMajor("1050", "USD")
+/// console.assert("10.50" === dollars)
+/// ```
+#[wasm_bindgen(js_name = minorToMajor)]
+pub fn minor_to_major_wasm(amt: &str, currency_code: &str) -> Result<String, JsError> {
+    girlmath::minor_to_major(amt, currency_code).map_err(|e| JsError::new(&e))
+}
+
+/// Converts an amount from one currency to another using a given exchange rate.
+///
+/// # Arguments
+/// * `payin_amt` - The amount in the pay-in currency as a string slice.
+/// * `payin_currency` - The pay-in currency code as a string slice (currently unused).
+/// * `payout_currency` - The pay-out currency code as a string slice.
+/// * `rate` - The exchange rate as a string slice.
+///
+/// # Returns
+/// the converted amount
+///
+/// # Examples
+/// ```js
+/// import girlmath from '@tbd54566975/girlmath'
+///
+/// const payoutAmt = girlmath.convert("3", "USD", "GHS", "12.10")
+/// console.assert("36.30" === payoutAmt);
+/// ```
+#[wasm_bindgen(js_name = convert)]
+pub fn convert_wasm(
+    payin_amt: &str,
+    _payin_currency: &str,
+    payout_currency: &str,
+    rate: &str,
+) -> Result<String, JsError> {
+    girlmath::convert(payin_amt, _payin_currency, payout_currency, rate)
+        .map_err(|e| JsError::new(&e))
+}
```

Comment:
`payin_currency` is being used here, so a little surprised to see the underscore prefix. Should this be used?

Reply:
yep the actual logic doesn't require it whatsoever. I kept `payin_currency` for readability sake and incase we need it for context later on (e.g. # of significant digits when dividing). 

re: readability. purely subjective and admittedly a bit weird, but i felt like looking at:

```js
girlmath.convert("3", "USD", "GHS", "12.10")
```

could easily be read as 
> convert 3 US to GHS with a rate of 12.10 GHS / 1 USD 

whereas

```js
girlmath.convert("3", "USD", "12.10")
```

would be read as:

> Convert 3 USD to ??? at a rate of 12.10 ??? / 1 USD

which felt like it was missing a part of the picture

Reply:
Ah, ok. That makes a bit more sense. Agreed that it does make it a bit more readable from the callsite, just thought it might have been an error in not using it when reading the actual code 😅